jarbas/core/models.py

```diff
@@ -11,9 +12,10 @@ def __init__(self, year, applicant_id, document_id):
         self.year = year
         self.applicant_id = applicant_id
         self.document_id = document_id
-        self.url = self.get_url()
+        self.exists_cache = None
```

Comment:
I'm fairly confused about this variable name - variable which may contain an Exception or a Bool value. Can't think about a good name to fit both types, maybe a good reason for refactor?